cirq/testing/consistent_qasm.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import warnings
-from typing import Any, List, Sequence
+from typing import Any, List, Sequence, Optional
```

Comment:
ordering

Reply:
on the basis that none of the imports are ordered in cirq and we don't have tooling to check for or support these, I'm going to not do any alphabetizing of imports :)

Reply:
Fair enough in general.

Maybe at least keep the alphabetic order where it pre-exists, e.g. here, and do whatever elsewhere?

Reply:
discussed elsewhere, I'm going to hold out for tooling to do this for me